### ISSUE TITLE
Breadcrumb icons are not accessible

### ISSUE THREAD

**Issue by djcelaya** [az-digital/arizona-bootstrap#2244]
The icon is not accessible because a pseudo-element is not a normal HTML element and cannot reliably have `aria-hidden="true"` applied to it. We should replace the icon-font Unicode character that is displaying this icon in CSS content with a background or mask that provides the same icon.
